src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -195,7 +209,7 @@ class LocalLambdaRunner {
         private readonly configuration: SettingsConfiguration,
         private readonly localInvokeArgs: LambdaLocalInvokeArguments,
         debugPort: number | undefined,
-        private readonly runtime: string,
+        private readonly _runtime: string,
```

Comment:
What's the point of this change?

Reply:
See Line 182

Reply:
No need to add an accessor here, just use `public readonly runtime: string`.